cinema/models.py

```diff
@@ -76,11 +90,16 @@ class Meta:
 
 class Ticket(models.Model):
     movie_session = models.ForeignKey(
-        MovieSession, on_delete=models.CASCADE, related_name="tickets"
+        MovieSession,
+        on_delete=models.CASCADE,
+        related_name="tickets"
     )
+
     order = models.ForeignKey(
-        Order, on_delete=models.CASCADE, related_name="tickets"
-    )
+        Order,
+        on_delete=models.CASCADE,
+        related_name="tickets")
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue here as well